pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDao.java

```diff
@@ -438,16 +438,14 @@ public Optional<KitRequestShipping> getKit(Long kitId) {
         return Optional.ofNullable((KitRequestShipping) results.resultValue);
     }
 
-    private SimpleResult deleteKitRequest(Connection conn, int kitRequestId) {
-        SimpleResult execResult = new SimpleResult();
+    private void deleteKitRequest(Connection conn, int kitRequestId) throws SQLException {
         try (PreparedStatement stmt = conn.prepareStatement(SQL_DELETE_KIT_REQUEST)) {
             stmt.setInt(1, kitRequestId);
-            execResult.resultValue = stmt.executeUpdate();
-        } catch (SQLException sqle) {
-            execResult.resultException = sqle;
+            int rowsAffected = stmt.executeUpdate();
+            if (rowsAffected == 0) {
+                logger.error("No kit request found with id: %d".formatted(kitRequestId));
```

Comment:
I didn't think this was a case for an exception. It is definitely something odd that we would want to know about, but since this is only called from test classes for cleaning their data, should the not existence of a kit stop the whole tearDown process in tests? I was not sure about that

Reply:
Did you observe this failing?
Test classes are a pretty controlled environment, so if this fails it probably indicates something wrong with your test.
Detecting a bad test, or a test failure, is more important than interrupting tear down. If the tear down fails silently, that's a problem, but in this case that will not happen.
You might want to add a "@visiblefortesting" annotation to this method.

Reply:
I see your point, and made the changes to throw here